likecoin/js/admin/likecoin_metabox.js

```diff
@@ -76,15 +128,57 @@ function onSubmitToISCN(e) {
   window.addEventListener('message', onISCNCallback, false);
 }
 
+async function onEstimateAndUploadArweave(e) {
+  e.preventDefault();
+  try {
+    const res = await jQuery.ajax({
+      type: 'POST',
+      url: `${wpApiSettings.root}likecoin/v1/posts/${wpApiSettings.postId}/arweave/estimate`,
+      method: 'POST',
+      beforeSend: (xhr) => {
+        xhr.setRequestHeader('X-WP-Nonce', wpApiSettings.nonce);
+      },
+    });
+    const { ipfsHash, LIKE, memo } = res;
+    if (LIKE === '0' || LIKE === 0) {
```

Comment:
I think using `arweaveId` exists is more robust, i.e. `res.arweaveId` and `res.ipfsHash` exists means we can skip upload

Reply:
true, will use arweave ID instead.